src/wallet/test/coinselector_tests.cpp

```diff
@@ -1182,7 +1182,14 @@ BOOST_AUTO_TEST_CASE(coin_grinder_tests)
             }
             return available_coins;
         });
-        BOOST_CHECK(res);
+        SelectionResult expected_result(CAmount(0), SelectionAlgorithm::CG);
+        for (int i = 0; i < 10; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < 10; ++i) {
```
Preference is given to `++i` according to `developer-notes.md`

Reply:
Thanks, done.  I did a copy paste from above (line 1177) which I guess should be changed also but not here.